packages/uswds-core/src/styles/settings/_settings-typography.scss

```diff
@@ -418,3 +418,7 @@ $theme-lead-font-family: "heading" !default;
 $theme-lead-font-size: "lg" !default;
 $theme-lead-line-height: 6 !default;
 $theme-lead-measure: 6 !default;
+
+// Margin
+$theme-heading-margin-top: 1.5em !default;
```

Comment:
Note: For reference, updating this value (with all global styles set to false) updated the margins in the following items:
- `* + .usa-display`
- `.usa-prose > * + h1` (through h6)
- `* + .usa-graphic-list__heading`
- `* + .usa-hero__heading`

Reply:
Cycling back to say that this comment was just to identify what was affected. I am resolving this because it is not necessary and can be ignored.